<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Taza de aprendizaje 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="taza_aprendizaje1">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.3563</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="taza_aprendizaje2">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.6765</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Taza de aprendizaje 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="taza_aprendizaje3">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.673442</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Taza de aprendizaje 3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="taza_aprendizaje4">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.24235632</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Taza de aprendizaje 4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="taza_aprendizaje5">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.46563</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Taza de aprendizaje 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aleatorio">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aleatorio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_manual">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>error</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="error_field">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.01</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="w0_field">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>240</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.141,0.366565,0.54654,0.2353225</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>w0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
